plasmapy/physics/tests/test_quantum.py

```diff
@@ -48,3 +60,37 @@ def test_deBroglie_wavelength():
 
     with pytest.raises(ValueError):
         deBroglie_wavelength(8*u.m/u.s, 'sddsf')
+
+# defining some plasma parameters for tests
+T_e = 1 * u.eV
+n_e = 1e23 * u.cm**-3
+# should probably change this to use unittest module
+# add tests for numpy arrays as inputs
+# add tests for different astropy units (random fuzzing method?)
+def test_thermal_deBroglie_wavelength():
+    r"""Test the thermal_deBroglie_wavelength function in quantum.py."""
+    lambda_dbTh = thermal_deBroglie_wavelength(T_e)
+    # test a simple case for expected value
+    assert np.isclose(lambda_dbTh.value,
+                      6.919367518364532e-10)
+    # testing returned units
+    assert lambda_dbTh.unit == u.m
+
+def test_Fermi_energy():
+    r"""Test the Fermi_energy function in quantum.py."""
+    energy_F = Fermi_energy(n_e)
+    # test a simple case for expected value
+    assert np.isclose(energy_F.value,
+                      1.2586761116196002e-18)
+    # testing returned units
+    assert energy_F.unit == u.J
+
+def test_Thomas_Fermi_length():
+    r"""Test the Thomas_Fermi_length function in quantum.py."""
+    lambda_TF = Thomas_Fermi_length(n_e)
+    # test a simple case for expected value
+    assert np.isclose(lambda_TF.value,
+                      5.379914085596706e-11)
```

Comment:
And likewise :) Basically, a "this is the expected value because..." either in the assert statement or in the docstring would be perfect.